### PR TITLE
Parse config in a more modular way

### PR DESCRIPTION
**Pull request by Quaqqer**

### CODE REVIEW

**Comment by Quaqqer** [comment]
A problem with this is that options are shown in commands for which they may not be relevant, but this seems like a somewhat good way to parse options.

File: src/nekontrol/interactive/cli.py [hunk 1]
```diff
@@ -1,12 +1,46 @@
...
+        @click.option(
+            "--ignore-debug/--no-ignore-debug",
+            default=None,
+            help="Ignore lines starting with `debug`",
+        )
+        @click.option(
+            "--force/--no-force",
+            default=None,
+            help="Submit even if sample verification fails",
+        )
```
Comment (sornas) [inline src/nekontrol/interactive/cli.py:26]: this means that all commands have a `--force`, yes? not sure that's what we want. especially when the help text talks about submitting.

Comment (sornas) [reply src/nekontrol/interactive/cli.py:26]: sorry i missed your comment before reviewing :x

**Comment by sornas** [comment]
> A problem with this is that options are shown in commands for which they may not be relevant, but this seems like a somewhat good way to parse options.

yeah like i said in the review that's a bit unfortunate. but overall this is better so we can fix that specifically later.

**Review by sornas — APPROVED** [review]

**Comment by sornas** [comment]
or did you want to fix it here?

**Pull request merged**